tests/read_pattern_test.py

```diff
@@ -0,0 +1,66 @@
+# content of read_pattern_test.py
+from safegraph_py_functions.safegraph_py_functions import read_pattern_single, read_pattern_multi
+import pytest
+import pandas as pd
+import pandas.util.testing as pdt
+import json
+import gzip
+
+
+### Files for analysis
+
+file_path = "tests/mock_data_v2020_09/weekly_demo.csv.gz"
+
+file_path_multi = "tests/mock_data_v2020_09/weekly_multi"
+
+### End files for analysis
+
+
+week1_path = "tests/mock_data_v2020_09/weekly_multi/weekly_demo1.csv.gz"
+week2_path = "tests/mock_data_v2020_09/weekly_multi/weekly_demo2.csv.gz"
+
+week1 = pd.read_csv(week1_path, dtype={'postal_code': str, 'phone_number': str, 'naics_code': str, 'latitude': float, 'longitude': float, 'poi_cbg': str, 'census_block_group': str,'primary_number': str}, compression='gzip')
```

Comment:
@Trippl7777 can save a little clutter by saving (on line 21)  a dict `sg_dtypes` where you define the dtypes then use this in both line 22 and line 23